test/test_pep518.py

```diff
@@ -24,7 +28,8 @@
 requires = [
     "setuptools>=42",
```

Comment:
```suggestion
    "setuptools >= 42",
```

We could add this, just to ensure this covers #392